src/plots/gl3d/layout/defaults.js

```diff
@@ -59,7 +59,7 @@ function handleGl3dDefaults(sceneLayoutIn, sceneLayoutOut, coerce, opts) {
     var bgcolor = coerce('bgcolor'),
         bgColorCombined = Color.combine(bgcolor, opts.paper_bgcolor);
 
-    var cameraKeys = Object.keys(layoutAttributes.camera);
+    var cameraKeys = ['up', 'center', 'eye'];
```

Comment:
Is that to improve perf :racehorse: ?

If so, why not, I think those `camera` attributes are pretty much :lock: ed forever.

Reply:
not 🐎 , just to avoid having to strip out `editType`